docs/backend.md

```diff
@@ -111,6 +111,19 @@ def baz(run, value: "var#foo"=42):
     return value
 ```
 
+Variable functions can use up to 4 CPU cores and 25 GB of RAM by default.
```

Comment:
This is slightly misleading because it implies that those limits are always set, whereas they're only used for automatically launched jobs from the listener. I'd reword this to something like:
```suggestion
To avoid accidentally running out of resources when processing runs automatically (i.e. triggered by migration or calibration during an experiment), the backend will launch its jobs for *non-cluster* variables with limits of 4 CPU cores and 25 GB of RAM by default. This only affects jobs launched automatically by the backend, running `amore-proto reprocess` manually or specifying `cluster=True` will still allow using the entire node.
```

Reply:
I was avoiding this topic for now in the interests of getting this merged, but...

I actually think that reprocessing should work by submitting a Slurm job too, and I'd like to work on that soon, but I think it's a bit more complicated, so I didn't want to hold this bit up. Having consistent rules for non-cluster variables seems like a good thing - you're less likely to write something that works when you test it and then fails on new runs, and I think it will make triggering reprocessing from the GUI (Qt or web) easier as well.  Plus it's less to explain. :slightly_smiling_face: 

I think we can figure something out to show the processing output 'live' in the terminal for `amore-proto reprocess`, either using Slurm's own `srun` command, or something like our [sfollow](https://github.com/European-XFEL/sfollow) wrapper.